test/config.test.js

```diff
@@ -24,7 +25,7 @@ describe('readConfig()', () => {
     mock(fsConfig);
 
     const result = readConfig();
-    expect(result.path).to.equal(dbPath);
+    expect(result.path).to.equal(path.normalize(dbPath));
```

Comment:
![image](https://github.com/MytsV/todolor/assets/89994710/c6773ec8-07a7-4ab3-9990-d03ed4479971)
Here test failing. Better return it back

Reply:
Oops

Reply:
My aim is off, I changed the wrong test 👽

Reply:
Thank you for the review! A fix will follow